src/jvmMain/kotlin/io/github/fourlastor/wilds_launcher/extensions/InputStreamCaptureExtension.kt

```diff
@@ -0,0 +1,13 @@
+package io.github.fourlastor.wilds_launcher.extensions
+
+import java.io.InputStream
+import java.io.InputStreamReader
+
+fun InputStream.capture(log: (String) -> Unit) {
+    val reader = InputStreamReader(this).buffered()
```

Comment:
You can use a convenience method which exists in the kotlin stdlib

```suggestion
    val reader = this.bufferedReader()
```